### ISSUE TITLE
I dont have the file but its saying i do

### ISSUE THREAD

**Issue by Doge121212** [camren-m/XboxWheelCompatibility#95]
![image_2023-12-31_164233852](https://github.com/cmumme/XboxWheelCompatibility/assets/155332440/21fcba90-65d0-46f4-9ffd-8fb987987b3f)

**Comment by bloatorola** [comment]
I am having the same problem with version 1.2.1 on W10 Pro 22H2

**Comment by Zariaa27** [comment]
Have you tried looking for it in the uninstall menu of the windows settings to see if you haven't had an instance of it before?

**Comment by bloatorola** [comment]
Yes, I had not installed any version of this software before trying 1.2.1. I ended up installing 1.2.0 with no issue
